integration_testing/features/coach/coach-activate-lesson.feature

```diff
@@ -4,35 +4,68 @@ Feature: Coach activates and deactivates lessons
   Background:
     Given I am signed in to kolibri as coach user
       And I am on *Coach - '<class>' > Plan > Lessons* page
-      And I see the lesson <lesson>
+      And I see the table of lessons which includes <lesson>
 
```

Comment:
To be fully thorough (and consistent with the 5. in the Reviewer guidance), let's add the learner-side check in the Gherkin too. I suggest we add a line here:

```And in the second browser or an incognito tab I am signed in as a learner on *Learn > Classes > '<class>'*```

Each of the below scenarios should have the check from the learner point of view at the end:
```
When I reload the browser as a learner
Then I see the lesson '<lesson>'
---- OR
When I reload the browser as a learner
Then I don't see the lesson '<lesson>'
```

It may not be fully Gherkin-kosher, but we already use this in other scenarios, and it makes the testing more thorough.

Reply:
Changes were pushed in last commit.